tools/src/main/java/org/neo4j/tools/dump/DumpLogicalLog.java

```diff
@@ -128,15 +128,24 @@ public LogVersionedStoreChannel next( LogVersionedStoreChannel channel ) throws
         LogEntryReader<ReadableClosablePositionAwareChannel> entryReader = new VersionAwareLogEntryReader<>();
 
         IOCursor<LogEntry> entryCursor = new LogEntryCursor( entryReader, logChannel );
-        TransactionLogEntryCursor transactionCursor = new TransactionLogEntryCursor( entryCursor );
-        try ( IOCursor<LogEntry[]> cursor = regex == null ? transactionCursor :
-                new FilteringIOCursor<>( transactionCursor, new TransactionRegexCriteria( regex, timeZone ) ) )
+        if ( regex == null )
         {
-            while ( cursor.next() )
+            while ( entryCursor.next() )
             {
-                for ( LogEntry entry : cursor.get() )
+                out.println( entryCursor.get().toString( timeZone ) );
+            }
+        }
+        else
+        {
+            TransactionLogEntryCursor transactionCursor = new TransactionLogEntryCursor( entryCursor );
+            try ( IOCursor<LogEntry[]> cursor = new FilteringIOCursor<>( transactionCursor, new TransactionRegexCriteria( regex, timeZone ) ) )
```

Comment:
Format, line too long. And while on it, line 127 could use a line break as well.